modules/local/msnbasexic/resources/users/bin/msnbasexic.R

```diff
@@ -0,0 +1,390 @@
+suppressPackageStartupMessages(library(MSnbase))
```

Comment:
As mentioned above, move it to `bin`
```suggestion
#!/usr/bin/env Rscript

suppressPackageStartupMessages(library(MSnbase))
```

Reply:
Already did it on the previous commit regarding https://www.nextflow.io/docs/latest/module.html#module-binaries